code/game/machinery/constructable_frame.dm

```diff
@@ -97,14 +97,13 @@
 	return FALSE
 
 /obj/structure/frame/wrench_act(mob/living/user, obj/item/tool)
+	. = NONE
 	switch(default_unfasten_wrench(user, tool, 4 SECONDS))
 		if(SUCCESSFUL_UNFASTEN)
 			return ITEM_INTERACT_SUCCESS
 		if(FAILED_UNFASTEN)
 			return ITEM_INTERACT_BLOCKING
 
-	return NONE
```

Comment:
This seems at tad un-necessary and as per the style guide prefer return over dot usage https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-